boards/xtensa/esp32/doc/index.rst

```diff
@@ -48,21 +48,56 @@ System requirements
 Prerequisites
 =============
 
-Two components are required in order to build this port: the `toolchain`_
-and the `SDK`_.  Both are provided by the SoC manufacturer.
+Some tools need to be installed on the computer before proceeding to the next steps.
+Follow the `installing prerequisites`_ instructions for your OS.
 
-The SDK contains headers and a hardware abstraction layer library
-(provided only as object files) that are required for the port to
-function.
+In order to build this port, the `SDK`_ and the `toolchain`_ need
+to be manually addressed. Both are provided by the SoC manufacturer.
+Cloning ESP_IDF is not necessary as it is automatically fetched in external modules.
 
-The toolchain is available for Linux, Windows, and Mac hosts and
-instructions to obtain and set them up are available in the ESP-IDF
-repository, using the toolchain and SDK links above.
+For the following environment setup, please check `set up the tools`_
+and `set up environment variables`_ guide to get specific OS configurations if needed.
 
 Set up build environment
 ========================
 
-With both the toolchain and SDK installed, the Zephyr build system must be
+Set ESP_IDF_PATH to external ESP_IDF folder:
+
+.. code-block:: console
+
+   # On Linux and macOS:
+   export ESP_IDF_PATH=<zephyr workspace top>/modules/hal/esp-idf"
+
+   # On Windows:
+   set ESP_IDF_PATH=<zephyr workspace top>\modules\hal\esp-idf"
```

Comment:
this should be hal_espressif now

Reply:
In fact, it is "espressif" now. `modules/hal/espressif`.

I will have to adress the path in some way due to the need of `./install.sh` and `source export.sh` within modules folder.